mitosheet/src/mito/components/footer/SheetTab.tsx

```diff
@@ -145,7 +145,7 @@ export default function SheetTab(props: SheetTabProps): JSX.Element {
 
                 if (props.tabIDObj.tabType === 'graph') {
                     // If opening a graph tab, close the cell editor 
-                    props.setEditorState(undefined)
+                    props.setEditorState(prevEditorState=>({type:"closed", editingMode:prevEditorState.editingMode}))
```

Comment:
I love this as a one liner -- want to move them all to this formatting? Also -- missing space after :'s